packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -144,6 +144,59 @@
     </SG.Item>
   </Shw::Grid>
 
+  <Shw::Text::H3>With different types of links as content</Shw::Text::H3>
```

Comment:
Probably we should showcase the same use cases also for the `Toast` component: https://hds-showcase-git-hds-2396-fix-alert-links-hashicorp.vercel.app/components/toast

Reply:
I added an example.